tests/integration/docker_test/chrome.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stable-slim
+
+# Install chrome driver
+RUN apt update -y && apt clean -y
+RUN apt install gnupg -y
+ENV CHROME_VERSION "google-chrome-stable"
```

Comment:
Why is this declared twice?

Reply:
Good point, I don’t know why so I can remove the duplicate line

Reply:
this was done here: https://github.com/drashland/sinco/pull/49/commits/1b89b5fbf726b2eec6eab88b068b3ecd32f7f99f